### PR TITLE
Updates for BB2.5-alpha.4

### PR DESCRIPTION
**Pull request by nancyhild**
Focuses mainly on updating the summary page (About This Release). Changes still need to be pushed down to existing articles.

### CODE REVIEW

**Comment by nancyhild** [comment]
Can be merged pending approval by Jamie.

**Review by Dannyholt — APPROVED** [review]

**Review by Dannyholt — APPROVED** [review]

**Review by codente — APPROVED** [review]

**Pull request merged**
